### PR TITLE
Datasets

### LINKED ISSUE #5: Data Cleaning & Prep for Databse
Go through the datasets and make sure they are easily parse able.

For the diversity dataset, there needs to be another row that holds some key that can be used to join with the other datasets.

### PR DESCRIPTION
**Pull request by asclines**
Closes #5 

For this review please focus on the [README](https://github.com/CodeSpaceHQ/SDA/blob/datasets/datasets/README.md) and let me know if those dataset schemes work with you..

Also note the filenames.

### CODE REVIEW

File: datasets/README.md [hunk 1]
```diff
@@ -1,14 +1,80 @@
...
+Dataset Filename: `income-data.csv`    
+UID: ZIPCODE
 
+| Attribute    | Type | Description |
+| ---          | ---  | ---         |
+| STATEFIPS    | Char | 01-56       |
+| STATE        | Char | Two-digit State abbreviation code |
+| ZIPCODE      | Char | |
+| AGI_STUB     | Num  | See below |
+| NUM_RETURNS  | Num  |  |
```
Comment (isaac-gs) [inline datasets/README.md]: Amount of space in the empty description box is different for this and ZIPCODE. Not sure if it really matters though

File: datasets/README.md [hunk 2]
```diff
@@ -1,14 +1,80 @@
...
+- 3 = $50,000 under $75,000
+- 4 = $75,000 under $100,000
+- 5 = $100,000 under $200,000
+- 6 = $200,000 or more
+
+
+> [Source](https://www.irs.gov/uac/soi-tax-stats-individual-income-tax-statistics-zip-code-data-soi)
+
+### Starbucks Locations Dataset
+_All the US Starbucks Locations_
```
Comment (isaac-gs) [inline datasets/README.md:30]: Did you cut out all of the non-US locations? Because the original was international

File: datasets/README.md [hunk 3]
```diff
@@ -1,14 +1,80 @@
...
+> [Source](https://www.kaggle.com/starbucks/store-locations)
+
+### Diversity Dataset
+_County Diversity Information_  
+Dataset Filename: `diversity.csv`    
+UID: COUNTY + STATE
+
+| Attribute    | Type | Description          |
+| ---          | ---  | ---                  |
+| COUNTY | Char |   |
```
Comment (isaac-gs) [inline datasets/README.md]: Inconsistent indentation

File: datasets/README.md [hunk 4]
```diff
@@ -1,14 +1,80 @@
...
+
+> [Source](https://github.com/kdallas2/diversity/blob/master/di.csv)
+
+### Location Dataset
+_Zipcodes, Cities, States & Counties_  
+Dataset Filename: `locations.csv`    
+
+| Attribute    | Type | Description          |
+| ---          | ---  | ---                  |
+| ZIPCODE | Num |   |
```
Comment (isaac-gs) [inline datasets/README.md]: Not sure if it matters. But inconsistent gap/indentation

**Review by isaac-gs — APPROVED** [review]
Looks good. Just a few questions

File: datasets/README.md (hunk 2 above)
Comment (asclines) [reply datasets/README.md:30]: Yes I cut them out

**Review by dlowrey — APPROVED** [review]
Looks good to me

**Comment by kevintaing86** [comment]
Should the diversity set have zip codes? I think if we just had county, it might be too generic?

**Comment by asclines** [comment]
@kevintaing86 unfortunately the dataset we have has the data by county not zip code. If you can find one that has zip codes instead, we can explore that route.

**Comment by kevintaing86** [comment]
No worries, I just didn't know off the top of my head if we had the zip code on that table or not. With our short deadline and busy schedules, I don't think it's in our best interest to dig for these datasets (it was a pain to look for what we have now).

**Pull request merged**